publisher/publisher.go

```diff
@@ -146,7 +146,7 @@ func main() {
 		}
 		cfg := lock.NewS3Config(
 			conf.awsLockBucket,
-			conf.awsLockBucket,
+			conf.awsRoleARN,
```

Comment:
why are we doing this change?

Reply:
that's already fixed on the main branch, I just merged from main so this should be gone